aup-previous.markdown

```diff
@@ -0,0 +1,154 @@
+**EXOSCALE ACCEPTABLE USE POLICY (EUSA) **
+
+Applicable starting October 12th, 2020
+
+[Previous version](https://github.com/exoscale/terms/blob/master/aup-previous.markdown)
+[Compare](https://github.com/exoscale/terms/commits/master)
+
+# 1. Introduction
+
+This Acceptable Use Policy ("AUP") describes prohibited use of the
+services provided by Akenes SA (hereafter "Exoscale") and activities
+that are not allowed on the Exoscale Services. The examples in
+this policy are not exhaustive. Exoscale may modify the AUP at any time
+and inform it's clients accordingly. If you violate the AUP, authorize
+or assist others to do so, Exoscale is entitled to suspend or terminate
+your account on the Exposcale Services as described in the
+Exoscale Terms and Conditions or Exoscale End User Service Agreement. 
+
+Capitalized terms used in this AUP and not otherwise defined herein shall have the meaning given in the Exoscale Terms and Conditions or Exoscale End User Service Agreement.
```

Comment:
Maybe put links to these documents here (or a reference if you want it to look "professional")?